test/Common/Include.hs

```diff
@@ -59,12 +58,13 @@ extensionOperations rootDrive = do
     where
 
     runTests parse file ext = do
-        let maybeFile = parse file
-        let maybeFileWithExt = parse (file ++ ext)
-        it ("Files " ++ file ++ " and " ++ (file ++ ext) ++ " are parsed successfully.") $
-            (isJust maybeFile, isJust maybeFileWithExt) `shouldBe` (True, True)
-        when (isJust maybeFile && isJust maybeFileWithExt) $
-            validExtensionsSpec ext (fromJust maybeFile) (fromJust maybeFileWithExt)
+        let maybePathFile = parse file
+        let maybePathFileWithExt = parse (file ++ ext)
+        case (maybePathFile, maybePathFileWithExt) of
+            (Just pathFile, Just pathFileWithExt) -> validExtensionsSpec ext pathFile pathFileWithExt
+            _ -> it ("Files " ++ file ++ " and/or " ++ (file ++ ext) ++ " should parse successfully.") $
+                     -- NOTE: Guaranteed to fail, but we are still doing this check to print information.
+                     show (maybePathFile, maybePathFileWithExt) `shouldBe` "(Just _, Just _)"
```

Comment:
You can do this with `expectationFailure <custom string here>`.

Reply:
Great, thanks, I missed that one! I didn't check Test.Hspec.Expectations docs because I didn't realize Test.Hspec re-exports it. Nice, this is explicit now.